src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:8080/" method="post">
+
+      <fieldset class="input-group">
+        <legend>Personal information</legend>
+
+        <label class="input-row">
+          Surname:
+          <input
+            name="surname"
+            type="text"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input-row">
```

Comment:
add one empty line between same level elements
```suggestion
        </label>
        
        <label class="input-row">
```

Reply:
fixed